config/samples/_v1alpha1_orchestrator_rc.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: rhdh.redhat.com/v1alpha1
+kind: Orchestrator
+metadata:
+  name: orchestrator-sample
+spec:
+  sonataflow:
+    images:
+      4.13: brew.registry.redhat.io/rh-osbs/iib:826480
```

Comment:
Shouldn't we be using production registry? This one is for testing/staging and it's internal to RH if I'm not mistaken.

Reply:
that RC sample, mostly for QE to test I think
And brew is mirroring internal registry, there is a step needed to get the access token but it's available from outside the VPN